### PR TITLE
SAK-40627 provide an error to user if autosave fails

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ samigo/samigo-app/src/webapp/js/saveForm.js  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4552313&bid=15313690)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ samigo/samigo-app/src/webapp/js/saveForm.js  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4552313&bid=15313690)

File: samigo/samigo-app/src/webapp/js/saveForm.js [hunk 1]
```diff
@@ -20,12 +21,11 @@ var doautosave = true;
...
 // have id's you'll need another way to identify them.
 //    disabledLinks is a list of triples, [id, onmouseup, onclick]
-
 var disabledLinks = [];
 
-function GetFormContent(formId, buttonName) {
-
+function GetFormContent() {
     try {
+        toggleSubmissionControls(false);
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/js/saveForm.js]: Looks like this comes from a previous patch? I reverted the behavior in #7562

File: samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp [hunk 2]
```diff
@@ -448,6 +448,13 @@ document.links[newindex].onclick();
 
   <f:verbatim><br/></f:verbatim>
 
+  <div id="autosave-conn-error" class="sak-banner-error tier1" style="display: none" role="alert">
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp]: Nice!

File: samigo/samigo-app/src/webapp/js/saveForm.js [hunk 3]
```diff
@@ -176,53 +209,15 @@ function SaveFormContentAsync(toUrl, formId, buttonName, updateVar, updateVar2,
...
-                ////al("microsoft");
-                result = new ActiveXObject("Microsoft.XMLHTTP");
-        } else {
-                throw new Error("failed to create XMLHTTPRequest");
-        }
-        return result;
-}
-
+    return saveok;
+}
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/js/saveForm.js]: EOF here.

**Pull request closed without merging**